ios/FluentUI/SegmentedControl/SegmentedControl.swift

```diff
@@ -373,32 +370,33 @@ open class SegmentedControl: UIView, TokenizedControlInternal {
             }
         }
 
-        let windowSafeAreaInsets = window.safeAreaInsets
-        let windowWidth = window.bounds.width - windowSafeAreaInsets.left - windowSafeAreaInsets.right
         if isFixedWidth {
-            if traitCollection.userInterfaceIdiom == .pad {
-                width = max(windowWidth / 2, Constants.iPadMinimumWidth)
-            } else {
-                width = windowWidth
+            if let window = window {
```

Comment:
Can you combine these two `if` statements?
```swift
if isFixedWidth, let window = window {
```

Reply:
That's not really the same logic, because right now if it fails the window check it doesn't go through the else of isFixedWidth.

Reply:
Ahhhhhh nested `if` statements make my brain go 🤯